packages/wallet-sdk/src/CoinbaseWalletProvider.test.ts

```diff
@@ -38,14 +38,6 @@ beforeEach(() => {
 });
 
 describe('Event handling', () => {
-  it('should emit connect on eth_requestAccounts request', async () => {
-    const connectListener = jest.fn();
-    provider.on('connect', connectListener);
-
-    await provider.request({ method: 'eth_requestAccounts' });
-    expect(connectListener).toHaveBeenCalledWith({ chainId: '0x1' });
-  });
```

Comment:
signer is responsible for this
![image](https://github.com/user-attachments/assets/3de7f499-8787-46c6-a122-a13a39a0e8a7)
![image](https://github.com/user-attachments/assets/140e6b39-dfb7-40c2-9b18-20de9071af08)